azure-communication-ui/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     ext {
-        call_library_version_name = '1.3.0'
+        call_library_version_name = '1.3.1-alpha.1'
```

Comment:
Should version say 1.3.1-beta.1 or even just 1.3.1?
If we plan to release it to public then it should be at least beta.
Or have you planned to update version in the release branch later?

Reply:
It should just say 1.3.1. 
Build version update is part of the release process in release branch. I'll update it to 1.3.1 after it gets merged. :)